pages/workspace/code-changes.mdx

```diff
@@ -3,16 +3,49 @@ layout: 'guide'
 title: 'Review Code Changes '
 ---
 
-Review the code changes affected by the branch by getting a good overview of added, deleted or updated events.
-
-The overview will display how many affected events are using Avo Codegen and who are not using it.
+The branch implement screen helps your team quickly and accurately implement tracking plan changes on a branch, allowing you start using the data or plan the next release. You can even share it with someone that's not part of your Avo workspace using the public sharing feature.
 
 <img
   src={require('../../images/code-changes/overview-screen.png')}
   width="100%"
   alt="Overview of code changes"
 />
 
+On the branch implement screen you see all events that are affected by your tracking plan changes, grouped by source. The event code snippet highlights exactly what has changed in the implementation of the event on the branch and makes it fast and easy for developers to understand what changes they need to make to successfully implement the branch. Your workspace members can comment directly no the events from the screen, quickly fleshing out any uncertainties about the changes. Your workspace members can comment directly on the events from the screen, quickly fleshing out any uncertainties about the changes. 
+
+## Sharing
+
+You can copy a link to the implementation instructions using the link (🔗) icon. It will include the current source, allowing you to tailor the link for each source that has to be implemented. If you want to focus on the implementation of a specific event, you can copy a link directly to it as well.
```

Comment:
```suggestion
You can copy a link to the branch code changes using the link (🔗) icon. It will include the current source, allowing you to tailor the link for each source that has to be implemented. If you want to focus on the implementation of a specific event, you can copy a link directly to each individual event as well.
```